examples/gpm-imerg-hhr/collection-template.json

```diff
@@ -0,0 +1,99 @@
+{
+  "type": "Collection",
+  "id": "gpm-imerg-hhr",
+  "stac_version": "1.0.0",
+  "description": "{{ collection.description }}",
+  "extent": {
+    "spatial": {
+      "bbox": [
+        [-180, -90, 180, 90]
+      ]
+    }
+  },
+  "links": [
+    {
+      "rel": "license",
+      "href": "https://lpdaac.usgs.gov/data/data-citation-and-policies/",
+      "title": "Public Domain"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/xarray-assets/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/scientific/v1.0.0/schema.json"
+  ],
+  "title": "gpm-imerg-hhr",
+  "providers": [
+    {
+      "name": "Development Seed",
+      "roles": [
+        "processor"
+      ],
+      "url": "https://developmentseed.org/"
+    },
+    {
+      "name": "Microsoft",
+      "roles": [
+        "host",
+        "processor"
+      ],
+      "url": "https://planetarycomputer.microsoft.com"
+    },
+    {
+      "name": "NASA",
+      "roles": [
+        "producer"
+      ],
+      "url": "https://gpm.nasa.gov/data/directory"
+    }
+  ],
+  "assets": {
+    "zarr-https": {
+      "href": "https://ai4edataeuwest.blob.core.windows.net/imerg/gpm-imerg-hhr.zarr",
+      "title": "gpm-imerg-hhr HTTPS Zarr root",
+      "description": "HTTPS URI of the gpm-imerg-hhr Zarr Group on Azure Blob Storage.",
+      "roles": [
+        "data",
+        "zarr",
+        "https"
+      ],
+      "type": "application/vnd+zarr",
+      "xarray:open_kwargs": {
+        "consolidated": true
+      }
+    },
+    "zarr-abfs": {
+      "href": "abfs://imerg/gpm-imerg-hhr.zarr",
+      "type": "application/vnd+zarr",
+      "description": "Azure Blob File System URI of the gpm-imerg-hhr Zarr Group on Azure Blob Storage for use with adlfs.",
+      "roles": [
+        "data",
+        "zarr"
+      ],
+      "xarray:storage_options": {
+        "account_name": "ai4edataeuwest"
+      },
+      "xarray:open_kwargs": {
```

Comment:
Does this also need the `use_cftime=True` option? If so, can you add it here?

We want to make sure that this works:

```python
collection = pystac.read_file("path/to/collection.json")
asset = planetary_computer.sign(pystac.assets["zarr-abfs"])
xr.open_zarr(asset.href, **asset.extra_fields["xarray:open_kwargs"])
```